assembl/static2/js/app/utils/translations.js

```diff
@@ -157,12 +157,22 @@ const Translations = {
         negativeTitle: "Contre",
         alternativeTitle: "Alternatif"
       },
+      createTopPost: {
+        positiveTitle: "Ajoutez votre point de vue en faveur du thème << %{ideaTitle} >>",
+        negativeTitle: "Ajoutez votre point de vue en défaveur du thème << %{ideaTitle} >>",
+        alternativeTitle: "Ajoutez votre point de vue alternatif sur le théme << %{ideaTitle} >>"
+      },
```

Comment:
wrong quotes, on linux
AltGR+w : «
and AltGR+x : »

Reply:
and `thème`, not `thème`

Reply:
Are you referring to `thème, ` vs `thème` here? I don't see a difference otherwise (unless you can read unicodes and the accented `è` is wrong.

Reply:
xD

Reply:
ok, I saw @n0izn0iz just fixed it